tests/test_bootstrap.py

```diff
@@ -469,3 +481,7 @@ async def test_setup_safe_mode_if_no_frontend(
         )
 
     assert "safe_mode" in hass.config.components
+    assert hass.config.config_dir == get_test_config_dir()
+    assert hass.config.skip_pip
+    assert hass.config.internal_url == "http://192.168.1.100:8123"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5716761)